lib/ansible/module_utils/crypto.py

```diff
@@ -26,12 +26,14 @@
 import hashlib
 
 
-def get_fingerprint(path):
+def get_fingerprint(path, passphrase):
     """Generate the fingerprint of the public key. """
 
     fingerprint = {}
 
-    privatekey = crypto.load_privatekey(crypto.FILETYPE_PEM, open(path, 'r').read())
+    privatekey = crypto.load_privatekey(crypto.FILETYPE_PEM,
+                                        open(path, 'r').read(),
```

Comment:
If you're going to make furhter changes, I'd also change this to 'rb' for potential forward compat.